examples/python_scripts/stacked_line_graphs.py

```diff
@@ -0,0 +1,85 @@
+import sys
+
+sys.path.append("/usr/gapps/spot/dev/hatchet-venv/x86_64/lib/python3.9/site-packages/")
+sys.path.append("/g/g20/hao3/thicket")
+sys.path.append("/usr/gapps/spot/dev/hatchet/x86_64/")
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import hatchet as ht
+import thicket as th
+import copy
+import re
+import argparse
+
+from IPython.display import display, HTML
+from glob import glob
+from hatchet import QueryMatcher
+
+
+def arg_parse():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--input_files", default="", type=str)
+    parser.add_argument("--groupby_parameter", default="", type=str)
+    parser.add_argument("--filter_prefix", default="", type=str)
+    parser.add_argument("--top_ten", default=False, type=bool)
+    parser.add_argument("--out_graphs", nargs="+")
```

Comment:
This one should have the `choices` argument, since you are looking for either `perc` or `total`

```suggestion
    parser.add_argument("--out_graphs", nargs="+", required=True, choices=["perc", "total"], type=str)
```